rollup.config.js

```diff
@@ -14,7 +14,7 @@ export default {
         {dest: 'dist/glamorous.es.js', format: 'es'},
         {dest: 'dist/glamorous.cjs.js', format: 'cjs'},
   ],
-  exports: 'default',
+  exports: 'named',
```

Comment:
It's the first time I use it as well.
As far as in understood, 'named' is necessary when you export default plus named